infra/release/release_note.md

```diff
@@ -1,28 +1,37 @@
-# Release Note 0.2.0
+# Release Note 0.3.0
 
 ## ONE-vscode
 
 ### ONE Explorer View
 
-- More commands are supported.
-  - [Open with Text Editor] command
-  - [Open Containing Folder] command
-  - [Collapse All] command
-- Improve [Create Config] command
+- TODO: Fill this part
 
 ### Toolchain View
 
-- Show Installable Toolchains on ToolchainView
-  - official release version
-  - nightly release version
-- Install Toolchain
-- Uninstall Toolchain
-- Show already-installed Toolchain
-- Support sudo command on Toolchain
+- TODO: Fill this part
+
+### Target Device View
+
+- Add Device list on local connection
+    - currently ubuntu 18.04 host PC and Tizen device added 
+- Show usable Executors under Device
 
 ### Cfg Editor
 
-- Editor is upgraded with whole new UI.
-- [Optimize] section is now supported.
-- [Quantize] section is now supported.
-- Bugfix
+- TODO: Fill this part
+
+### Circle Graph Viewer
+
+- TODO: Fill this part
```

Comment:
- Provides graphical view of circle model
- Powered by famous [netron](https://github.com/lutzroeder/netro) source code